packages/alfa-style/test/node/predicate/has-transparent-background.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import { h } from "@siteimprove/alfa-dom";
+import { test } from "@siteimprove/alfa-test";
+
+import { Device } from "@siteimprove/alfa-device";
+
+import { hasTransparentBackground } from "../../../src/element/predicate/has-transparent-background";
+
+const device = Device.standard();
+
+test("hasTransparentBackground() fails on image which is not transparent", (t) => {
+  const target = <img src="foo.jpg" />;
+
+  h.document([target]);
+
+  t.deepEqual(hasTransparentBackground(device)(target), false);
+});
+
+test("<div> has a color background", (t) => {
```

Comment:
```suggestion
test("hasTransparentBackground() returns false on element with a background color", (t) => {
```
Test names should be meaningful without context. If the test fails, this name is the first thing that will be printed on the terminal, so the first thing we read to figure out where (and why) the test fails.
* frontloading the tested function helps locating the problem ("<div> has a color background" doesn't help understanding in which file this is, or which function we tested).
* Trimming information to relevant level (It doesn't matter that this is a `<div>` and not a `<span>`, it only matters that it is an element).
* The predicate itself doesn't pass or fail. It returns `true` or `false`, but that is neither a success or failure in itself.